docs: rewrite Quick Start as short developer guide (CTORNDGAIN-1301) (#82)
griddynamics/rosetta: QUICKSTART.md

> # Quick Start — your first session with Rosetta

Reviewer: ```suggestion
# Quick Start

> [!CAUTION]
> You must receive a prior approval from your manager and company to use it.

> [!WARNING]
> Use **Sonnet 4.6**, **GPT-5.3-codex-medium**, **gemini-3.1-pro** or better models. Avoid Auto model selection.

> [!NOTE]
> Rosetta is designed to never use or see data or IP.
> Instead it uses inversion of control, by providing a "menu" to AI coding agents.

```

> **Who is this for?** You've just installed Rosetta and want to do your first real task.
> **When should I read this?** Right after install. For install steps, see [README](README.md) (Get started section) or [PLUGINS.md](PLUGINS.md). For troubleshooting setup itself, see [TROUBLESHOOTING.md](TROUBLESHOOTING.md).

Reviewer: ```suggestion
**Who is this for?** New users setting up Rosetta for the first time.
**When should I read this?** When you want to go from zero to a working setup.
```

---

> 

Reviewer: ```suggestion
## Install Rosetta

We recommend installing Rosetta using [PLUGINS](PLUGINS.md).
If AI coding agent does not support plugins, use [MCPs](MCPs.md).

---

```

## 1. Open the right folder

Open the **actual project repo** in your IDE — not the parent folder that contains it. Rosetta initializes whatever folder Claude is running in, so opening a parent directory scatters Rosetta files into the wrong place.

Quick check: in the IDE's terminal, run `pwd`. The output should be your project root (where your `package.json`, `pom.xml`, `pyproject.toml`, etc. lives), not its parent.

> ## 2. Start Claude and verify Rosetta loaded

Reviewer: Requires refactoring. This is not correct.
We give a single link to QuickStart.md to anybody to start to use ANY tool and installation. Including next steps.

Reviewer: MCPs.md contains Steps 1 Connect, 4 (non-optional), 2 (verify)
PLUGINS.md contains Steps 1 Install (each option in collapsed section) and 2 (verify)

Step 3 from MCPs.md goes here as second step

Next Steps from PLUGINS.md moved here + link to USAGE_GUIDE.md

Existing Next Steps renamed to Links 

All warnings in all files go in the very first section
